### PR TITLE
sidecar updates

### PR DESCRIPTION
**Pull request by nitesh3108**
# Description
1. Latest sidecar updates.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/350|

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Installed driver and performed sanity, results are passing.

image

### CODE REVIEW

File: dell-csi-helm-installer/verify-csi-isilon.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@
 #
 # verify-csi-isilon method
 function verify-csi-isilon() {
-  verify_k8s_versions "1.21" "1.24"
```
Comment (randeepsharma) [inline dell-csi-helm-installer/verify-csi-isilon.sh:14]: This would be needed for MKE 3.5.x

**Comment by randeepsharma** [comment]
Please make associated doc changes as well.

File: dell-csi-helm-installer/verify-csi-isilon.sh (hunk 1 above)
Comment (nitesh3108) [reply dell-csi-helm-installer/verify-csi-isilon.sh]: Added back 1.21 support

**Review by bpjain2004 — DISMISSED** [review]

**Comment by randeepsharma** [comment]
k8s 1.21 file is still deleted. Need to restore it and update side cars.

**Comment by nitesh3108** [comment]
> k8s 1.21 file is still deleted. Need to restore it and update side cars.

Will not update sidecars for 1.21

**Review by randeepsharma — APPROVED** [review]

**Review by Sakshi-dell — COMMENTED** [review]
need to test the changes with health monitor sidecar enabled in values.yaml

[root@rsvm-10-247-66-81 ~]# k describe po isilon-controller-7db4b4fbff-wbp4p -n isilon  | grep Image | grep -v ID
    Image:         k8s.gcr.io/sig-storage/csi-resizer:v1.5.0
    Image:         k8s.gcr.io/sig-storage/csi-attacher:v3.5.0
    Image:         k8s.gcr.io/sig-storage/csi-external-health-monitor-controller:v0.6.0
    Image:         k8s.gcr.io/sig-storage/csi-provisioner:v3.2.1
    Image:         k8s.gcr.io/sig-storage/csi-snapshotter:v6.0.1

**Review by Sakshi-dell — APPROVED** [review]

**Comment by nitesh3108** [comment]
> need to test the changes with health monitor sidecar enabled in values.yaml

isilon-controller-7db4b4fbff-wbp4p   6/6     Running   0          2m50s

[root@rsvm-10-247-66-81 ~]# k describe po isilon-controller-7db4b4fbff-wbp4p -n isilon  | grep Image | grep -v ID
    Image:         k8s.gcr.io/sig-storage/csi-resizer:v1.5.0
    Image:         k8s.gcr.io/sig-storage/csi-attacher:v3.5.0
    Image:         k8s.gcr.io/sig-storage/csi-external-health-monitor-controller:v0.6.0
    Image:         k8s.gcr.io/sig-storage/csi-provisioner:v3.2.1
    Image:         k8s.gcr.io/sig-storage/csi-snapshotter:v6.0.1

**Pull request merged**